### PR TITLE
2.5 networking frontend

### LINKED ISSUE #70: #2.5 Network Page
As a user, I want to be able to access a network page where I can view my connections, send, accept, and reject invitations, so that I can grow my network connections.

Story Points: 1
MoSCoW: Must
Business Value: XL
Risk: Low

Acceptance Criteria
- Upon landing on the website, login with your credentials
- Go to the network page by clicking the "Network" logo on the navigation bar

Child of #23

### PR DESCRIPTION
**Pull request by CHADBUCK13**
Added network frontend and fixed navbar/footer. Made small refactor to make isUserAuthenticated easier to use. Added network page to controller.

This PR closes #70

### CODE REVIEW

File: synapsis/src/main/java/com/soen/synapsis/appuser/AppUser.java [hunk 1]
```diff
@@ -92,6 +96,15 @@ public void setRole(Role role) {
         this.role = role;
     }
 
+    public static boolean isUserAuthenticated() {
```
Comment (JamesPartsafas) [inline synapsis/src/main/java/com/soen/synapsis/appuser/AppUser.java:99]: This is a good change.

**Review by JamesPartsafas — APPROVED** [review]

File: synapsis/src/main/resources/templates/pages/network.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                        <div class="font-sans text-base font-light mt-2 ml-4">Tanya Leduc</div>
+                        <div class="font-sans text-sm font-light mt-1 ml-4">Craigstutors</div>
+                        <div class="mx-auto mt-6 w-24">
+                            <button class="btn btn-secondary font-sans text-sm w-24 h-9 p-0">Add
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
```
Comment (janong24) [inline synapsis/src/main/resources/templates/pages/network.html:125]: Please remove the suggested section as discussed. Thanks!

**Review by janong24 — CHANGES_REQUESTED** [review]
Please remove the suggested section as discussed, thanks!

**Review by janong24 — APPROVED** [review]

**Comment by janong24** [comment]
All changes have been made. PR good to merge.

**Pull request merged**